js/app/modules/card/media.js

```diff
@@ -58,12 +58,12 @@ const Media = new Module.Class({
             attributions.push(this.model.copyright_holder);
         this.set_label_or_hide(this._attribution_label, attributions.map((s) => {
             return s.split('\n')[0];
-        }).join(' - ').toUpperCase());
+        }).join(' - '));
         this._attribution_button.visible = this._attribution_label.visible;
         this._attribution_button.connect('clicked', this._show_license_in_external_viewer.bind(this));
-
-        this._separator.visible = this._attribution_label.visible &&
-                                  this._caption_label.visible;
+        if (this._caption_label.visible && this._attribution_label.visible) {
```

Comment:
I'm not sure I understand what you mean by the container of a class. The selector `.bemclass.bemclass--modifier1.bemclass--modifier2` will select an element with both modifiers applied to it at once. It is indeed similar to what you were suggesting, the differences are splitting it into 2 modifiers and using the `--` BEM convention for modifiers.